{
  "name": "wait-for-throwable",
  "version": "1.0.0",
  "description": "Simple utility to retry an erroring function until it succeeds",
  "main": "wait-for-throwable.js",
  "files": [
    "wait-for-throwable.js"
  ],
  "dependencies": {},
  "devDependencies": {
    "chai": "^4.2.0",
    "eslint": "^5.16.0",
    "mocha": "^8.1.3",
    "nyc": "^15.1.0",
    "safe-await": "^1.0.0",
    "sinon": "^9.0.3"
  },
  "scripts": {
    "test": "nyc mocha test.js"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/catdad/wait-for-throwable.git"
  },
  "author": "Kiril Vatev <vatev.1@gmail.com>",
  "license": "ISC",
  "bugs": {
    "url": "https://github.com/catdad/wait-for-throwable/issues"
  },
  "homepage": "https://github.com/catdad/wait-for-throwable#readme",
  "engines": {
    "node": ">=10.0.0"
  },
  "keywords": [
    "wait",
    "waitFor",
    "waitUntil",
    "test",
    "promise",
    "mocha",
    "jest",
    "assert",
    "assertion",
    "expect",
    "chai"
  ]
}
